scripts/advisor/run_advisor.py

```diff
@@ -111,16 +96,17 @@ def run_with_advisor(path, output, name, exec_args, advisor_home, plot):
     ]
     advisor_survey = [
         '-collect survey',
-        '-start-paused',
         '-run-pass-thru=--no-altstack',  # Avoids `https://software.intel.com/en-us/vtune-amplifier-help-error-message-stack-size-is-too-small`  # noqa
         '-strategy ldconfig:notrace:notrace',  # Avoids `https://software.intel.com/en-us/forums/intel-vtune-amplifier-xe/topic/779309`  # noqa
         '-start-paused',  # The generated code will enable/disable Advisor on a loop basis
     ]
     advisor_flops = [
         '-collect tripcounts',
+        '-enable-cache-simulation',
```

Comment:
remind me what is this for?

Reply:
From my understanding, not setting this flag means that Advisor does not take into account which level of the cache hierarchy data is being moved from. This means that it does not generate specific data about which level of the hierarchy is providing bottlenecks to loop performance. Setting the flag allows Advisor to extrapolate information it collects by simulating different kinds of data transfer from different caches to the program. Perhaps we do not currently need this information although in testing both George and I used this flag for our roofline models.

The post on the official Intel forums explaining this is: https://software.intel.com/content/www/us/en/develop/articles/integrated-roofline-model-with-intel-advisor.html

Reply:
Yes, we would like to keep it, although not always useful for the overhead it has. I suggest we keep it now. I do not remember whether disabling that option... plots or not....cache level lines on roofline... lemme check...